index.ts

```diff
@@ -206,3 +209,13 @@ export type RejectionCode = Variant<{
     CanisterError: null;
     Unknown: null;
 }>;
+
+export type StableMemoryError = Variant<{
+    OutOfMemory: null;
+    OutOfBounds: null;
+}>;
+
+export type StableGrowResult = Variant<{
+    ok: nat32;
+    err: StableMemoryError;
+}>;
```

Comment:
@lastmjs just for learning's sake, why can't we do `Result<nat32, StableMemoryError>`? I see that you've commented out those types so I'm assuming they don't work for some reason. Otherwise that seems more elegant to me.

Reply:
@lastmjs says he doesn't remember why it's commented out, its a valid concern and we will address it later if we reintroduce the Result type, but for right now it's okay